### PR TITLE
Enable web test cases

### PR DESCRIPTION
**Pull request by rjkip**

### CODE REVIEW

**Comment by rjkip** [comment]
:eyeglasses: @DRvanR

File: app/config/config_test.yml [hunk 1]
```diff
@@ -20,8 +20,25 @@ doctrine:
...
+monolog:
+    handlers:
+        main_graylog:
+            type: test
+        main_logfile:
+            type: test
+        graylog:
+            type: test
+        console:
+            type:   test
```
Comment (DRvanR) [inline app/config/config_test.yml]: bit too much spacing :wink:

**Comment by DRvanR** [comment]
:+1:

**Comment by rjkip** [comment]
Changed `test` handler to `null`. The test handler aggregates the records, while the null handler just discards them.

**Pull request merged**